packages/design-system/src/styles/settings/variables/_build.scss

```diff
@@ -1,6 +1,6 @@
 // Paths
-$font-path: './fonts' !default;
-$image-path: './images' !default;
+$font-path: '../fonts' !default;
+$image-path: '../images' !default;
```

Comment:
Can you talk about this change here? I reverted back to the single `./` and didn't see a difference. What is this change fixing?

Reply:
The `../` fixes those error msgs on Website when viewing the dev tool's console 
![Screen Shot 2020-07-27 at 10 15 18 AM](https://user-images.githubusercontent.com/20322880/88552880-99f18900-cff2-11ea-8962-0420a79365dc.png)

Reply:
as the `src/font` & `src/image` folders are on the same level as `src/styles`

Reply:
I'll check again, looks like maybe it did not fix that.

Reply:
Checked - yes, it fixed the error msgs (I forgot to yarn build again after changing the variables).